examples/flutter_gallery/ios/Runner/Info.plist

```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
-	<string>Flutter Gallery</string>
+	<string>Design Lab</string>
```

Comment:
Does Flutter Design Lab not fit on the springboard?

Reply:
Nope